src/client/app/components/maps/EditMapModalComponent.tsx

```diff
@@ -0,0 +1,156 @@
+	const dispatch: AppDispatch = useAppDispatch();
+	const [nameInput, setNameInput] = useState(map.name);
+	const [noteInput, setNoteInput] = useState(map.note || '');
+	const [circleInput, setCircleInput] = useState(map.circleSize.toString());
+	const [displayable, setDisplayable] = useState<DisplayableType>(map.displayable);
+
+	const intl = useIntl();
+
+	const handleSave = () => {
+		const updatedMap = {
+			...map,
+			name: nameInput,
+			note: noteInput,
+			circleSize: parseFloat(circleInput),
+			displayable: displayable as DisplayableType
+		};
+		dispatch(editMapDetails(updatedMap));
+		dispatch(submitEditedMap(updatedMap.id));
+		handleClose();
+	};
+
+	const handleDelete = () => {
+		const consent = window.confirm(intl.formatMessage({ id: 'map.confirm.remove' }, { name: map.name }));
+		if (consent) {
+			dispatch(removeMap(map.id));
+			handleClose();
+		}
+	};
+
+	const handleCalibrationSetting = (mode: CalibrationModeTypes) => {
+		dispatch(setCalibration(mode, map.id));
+		handleClose();
+	};
+
+	const toggleCircleEdit = () => {
+		const regtest = /^\d+(\.\d+)?$/;
+		if (regtest.test(circleInput) && parseFloat(circleInput) <= 2.0) {
+			setCircleInput(circleInput);
+		} else {
+			showErrorNotification(intl.formatMessage({ id: 'invalid.number' }));
+		}
+	};
+
+	return (
+		<Modal isOpen={show} toggle={handleClose}>
+			<ModalHeader toggle={handleClose}>
+				<FormattedMessage id="edit.map" />
+			</ModalHeader>
+			<ModalBody>
+				<Form>
+					<FormGroup>
+						<Label for="mapName"><FormattedMessage id="map.name" /></Label>
+						<Input
+							id="mapName"
+							value={nameInput}
+							onChange={e => setNameInput(e.target.value)}
+						/>
+					</FormGroup>
+					<FormGroup>
+						<Label for='mapDisplayable'><FormattedMessage id='map.displayable' /></Label>
+						<Input
+							id='mapDisplayable'
+							name='mapDisplayable'
+							type='select'
+							value={displayable}
+							onChange={e => setDisplayable(e.target.value as DisplayableType)}
+						>
+							{Object.keys(DisplayableType).map(key => (
+								<option value={key} key={key}>
+									{intl.formatMessage({ id: `map.is.${key}` })}
+								</option>
+							))}
+						</Input>
+					</FormGroup>
+					<FormGroup>
+						<Label for="mapCircleSize"><FormattedMessage id="map.circle.size" /></Label>
+						<Input
+							id="mapCircleSize"
+							type='number'
+							value={circleInput}
+							onChange={e => setCircleInput(e.target.value)}
+							invalid={parseFloat(circleInput)<0}
+							onBlur={toggleCircleEdit}
+						/>
+					</FormGroup>
+					<FormGroup>
+						<Label for="mapNote"><FormattedMessage id="note" /></Label>
+						<Input
+							id="mapNote"
+							type="textarea"
+							value={noteInput}
+							onChange={e => setNoteInput(e.target.value.slice(0,30))}
+						/>
+					</FormGroup>
+				</Form>
+				<div>
+					<Label><FormattedMessage id="map.filename" /></Label>
+					<Input
+						id='mapFilename'
```

Comment:
As above for file, standardize with card view.

I also want to say that I personally find the spacing to be confusing. I think the one line for title/info will help but also putting a space before this would help separate it from file.

Reply:
The next separator resolves the spacing issue but not the placement of items together.